spec/features/authentication/visitor_creates_an_account_spec.rb

```diff
@@ -1,43 +1,47 @@
 require "rails_helper"
 
-describe User, type: :feature do 
-  describe "A visitor" do 
-    it "can register as a user with correct field inputs" do 
+describe User, type: :feature do
+  describe "A visitor" do
+    it "can register as a user with correct field inputs" do
+      email = "#{(0..1000).to_a.sample}@email.com"
+      phone_number = "#{(0..1000).to_a.sample(10)}"
+
       visit root_path
       click_on "Create Account"
       expect(current_path).to eq(new_user_path)
 
       fill_in "First Name", with: "First Name"
       fill_in "Last Name", with: "Last Name"
-      fill_in "Email", with: "Email"
-      fill_in "Phone Number", with: "Phone"
+      fill_in "Email", with: email
+      fill_in "Phone Number", with: phone_number
       fill_in "Password", with: "Password"
       fill_in "Confirm Password", with: "Password"
-      
+
       click_on "Sign Up"
 
-      user = User.find_by(first_name: "First Name")
+      user = User.last
+
       expect(current_path).to eq(user_path(user.id))
       expect(page).not_to have_content("Create Account")
       expect(page).to have_content("First Name")
       expect(page).to have_content("Last Name")
-      expect(page).to have_content("Email")
-      expect(page).to have_content("Phone")
+      expect(page).to have_content(email)
+      expect(page).to have_content(phone_number)
     end
 
```

Comment:
Ohhhh I see what you're saying. I think making an existing user with Faker/Factory then just doing the fill_in steps is a fine way to go.